bpython/repl.py

```diff
@@ -439,7 +439,8 @@ def __init__(self, interp, config):
             duplicates=config.hist_duplicates, hist_size=config.hist_length
         )
         self.s_hist = []
-        self.history = []
+        self.history = []  # History of commands
```

Comment:
how about "commands executed since beginning of session," this comment isn't helping me much now.